core/domain/feedback_services.py

```diff
@@ -266,7 +283,7 @@ def enqueue_feedback_message_email_task(user_id):
 
 
 def get_feedback_message_references(user_id):
-    model = feedback_models.UnsentFeedbackEmailModel.get(user_id, strict=True)
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
```

Comment:
I don't think that its going to happen. Because if a task is added to queue then model must have been created and if model is created then it must exist when executing handler function and above function is part used only in handler function, so model must exist. Even if lets say we do not have model in such situations, we certainly do not want it to end noisily, so if we return [] then email will still be sent but it will not have feedback in email body. So we also need a check in `email_manager.send_feedback_message_email`, that if it gets an empty dict, just do not send email and return from it. 
Does that fix sound good, although I still do not think we may need this. We can skip it altogether.